simapp/codec.go

```diff
@@ -15,6 +16,7 @@ type AppCodec struct {
 	amino *codec.Codec
 
 	Params       *params.Codec
+	Slashing     *slashing.Codec
```

Comment:
I do think we should have something more generic when modules don't actually do anything special with their codec. What do you think @alexanderbez ?

Reply:
this file should be removed as the new design does not contain this struct. you can see here: https://github.com/cosmos/cosmos-sdk/blob/08502d6d98bff3872916f8b69cde2d255351d3fb/simapp/codec/codec.go